content/academy/homepage_content.json

```diff
@@ -0,0 +1,33 @@
+{
+    "Beginner courses": [
+        {
+            "title": "Web scraping for beginners",
+            "link": "web-scraping-for-beginners",
```

Comment:
These links will not work on `developers.apify.com/academy` (the homepage) unless we change the links here to a format: `academy/[article-path]`. Or we hardcode the academy prefix to the `CardItem` on web. I would go with the first option. What do you think? @mnmkng @m-murasovs

Reply:
I think I had the `academy` there before, because intuitively it made sense like this, but it did not work locally. But you did some changes to the link property inside the other PR so this is a follow up to that?

Reply:
Yes I updated the links logic in this commit. It should be working if once you update the links here to the proposed format

Reply:
done